sqlglot/schema.py

```diff
@@ -96,6 +96,27 @@ def supported_table_args(self) -> t.Tuple[str, ...]:
 
         return self._supported_table_args
 
+    def normalize(self) -> None:
+        schema = self.mapping
+        flattened_schema = flatten_schema(schema, depth=dict_depth(schema) - 1)
+
+        normalized_mapping: t.Dict = {}
+        for keys in flattened_schema:
+            columns = _nested_get(schema, *zip(keys, keys))
+            assert columns is not None
+
+            for column_name, column_type in columns.items():
+                column = exp.to_identifier(column_name)
```

Comment:
I don't think this is right.

I think we want `parse_one(column_name, into=exp.Identifier)`

Reply:
Why do we need to parse this? Did I miss something?

Reply:
`to_identifier` assumes the string is the identifier and adds quotes if necessary.

```python
exp.to_identifier('"S"').sql()
# '""S""'

parse_one('"S"', into=exp.Identifier).sql()
# '"S"'
```